edward/client.go

```diff
@@ -81,7 +81,14 @@ func (c *Client) Start(names []string, skipBuild bool, tail bool, noWatch bool,
 	return nil
 }
 
-func (c *Client) Restart(names []string, skipBuild bool, tail bool, noWatch bool, exclude []string) error {
+func (c *Client) Restart(names []string, noPrompt bool, skipBuild bool, tail bool, noWatch bool, exclude []string) error {
+
+	// Prompt user to confirm the restart
+	if !noPrompt {
```

Comment:
If you're restarting all services, it makes sense to have this confirmation, but if you've specified a single service, it's pretty clear you know what you're doing.

Perhaps we could only show ask for confirmation when a certain number of services are being restarted? Or no services were specified at the command line?